lib/array.rb

```diff
@@ -0,0 +1,35 @@
+class Array
+  def faisal_inject (arg = 'empty', symbol = nil)
```

Comment:
Assignment Branch Condition size for faisal_inject is too high. [19.34/15]<br>Do not put a space between a method name and the opening parenthesis.